testsuite/features/step_definitions/retail_steps.rb

```diff
@@ -91,47 +91,38 @@ def compute_kiwi_profile_version(host)
   end
 end
 
-When(/^I enable repositories before installing branch server$/) do
+When(/^I (enable|disable) repositories during branch server installation$/) do |action|
   os_version = $proxy.os_version
   os_family = $proxy.os_family
 
   # Distribution
-  repos = 'os_pool_repo os_update_repo'
-  log $proxy.run("zypper mr --enable #{repos}")
+  repos = 'os_pool_repo os_update_repo testing_overlay_devel_repo'
+  log $proxy.run("zypper mr --#{action} #{repos}")
 
-  # Server Applications
+# Server Applications, proxy product and modules, proxy devel
   if os_family =~ /^sles/ && os_version =~ /^15/
-    repos = 'module_server_applications_pool_repo module_server_applications_update_repo'
-    log $proxy.run("zypper mr --enable #{repos}")
-  end
-end
-
-When(/^I disable repositories after installing branch server$/) do
-  os_version = $proxy.os_version
-  os_family = $proxy.os_family
-
-  # Distribution
-  repos = 'os_pool_repo os_update_repo'
-  log $proxy.run("zypper mr --disable #{repos}")
+    repos = 'proxy_module_pool_repo proxy_module_update_repo' \
+           'proxy_product_pool_repo proxy_product_update_repo' \
+           'server_devel_releasenotes_repo server_devel_repo' \
+           'module_server_applications_pool_repo module_server_applications_update_repo'
```

Comment:
missing `testing_overlay_devel_repo` ?

Reply:
Yeah, seems it's missing indeed. BTW, this list of repos is hard to read FMPOV.

I'd rather have one repo for file, but maybe it's just a matter of taste.

Reply:
Ah, line 99, ok.

Reply:
it's above, we missed it